### Importando los dataset hogares_viviendas_superficie.csv

In [12]:
import pandas as pd


#df = pd.read_csv("hogares_viviendas_superficie.csv")

df = pd.read_csv('hogares_viviendas_superficie.csv', index_col=0, encoding='latin-1')


### Visualizacion de los primeros 5 registros del DataFrame

In [13]:
df.head()

provincia  hogares  viviendas_particulares  \
provincia_id                                                     
2             Capital Federal  1150134                 1423973   
6                Buenos Aires  4789484                 5377786   
10                  Catamarca    96001                  113634   
14                    Córdoba  1031843                 1232211   
18                 Corrientes   267797                  292644   

              viviendas_particulares_habitadas  superficie_km2  
provincia_id                                                    
2                                      1082998             200  
6                                      4425193          307571  
10                                       89376          102602  
14                                      978553          165321  
18                                      248844           88199

### Agregando una nueva columna con un campo nuevo "densidad" (hogares/superficie)

In [15]:
df['densidad'] = df['superficie_km2']/df['hogares']*100

In [16]:
df.head()

provincia  hogares  viviendas_particulares  \
provincia_id                                                     
2             Capital Federal  1150134                 1423973   
6                Buenos Aires  4789484                 5377786   
10                  Catamarca    96001                  113634   
14                    Córdoba  1031843                 1232211   
18                 Corrientes   267797                  292644   

              viviendas_particulares_habitadas  superficie_km2    densidad  
provincia_id                                                                
2                                      1082998             200    0.017389  
6                                      4425193          307571    6.421798  
10                                       89376          102602  106.875970  
14                                      978553          165321   16.021914  
18                                      248844           88199   32.935022

### Cada hogar, en promedio, tiene una superficie de 0.017389 km2 (17.389 m2)

### Eliminar columnas innecesarias.

In [17]:
df = df.drop(columns=['viviendas_particulares' , 'viviendas_particulares_habitadas'])

In [18]:
df

provincia  hogares  superficie_km2     densidad
provincia_id                                                           
2                 Capital Federal  1150134             200     0.017389
6                    Buenos Aires  4789484          307571     6.421798
10                      Catamarca    96001          102602   106.875970
14                        Córdoba  1031843          165321    16.021914
18                     Corrientes   267797           88199    32.935022
22                          Chaco   288422           99633    34.544175
26                         Chubut   157166          224686   142.960946
30                     Entre Ríos   375121           78781    21.001490
34                        Formosa   140303           72066    51.364547
38                          Jujuy   174630           53219    30.475291
42                       La Pampa   107674          143440   133.216933
46                       La Rioja    91097           89680    98.444515
50                        Mendoza   494841          148827    30.075721
54                       Misiones   302953           29801     9.836839
58                        Neuquén   170057           94078    55.321451
62                      Río Negro   199189          203013   101.919785
66                          Salta   299794          155488    51.864947
70                       San Juan   177155           89651    50.605967
74                       San Luis   126922           76748    60.468634
78                     Santa Cruz    81796          243943   298.233410
82                       Santa Fe  1023777          133007    12.991794
86            Santiago del Estero   218025          136351    62.539158
90                        Tucumán   368538           22524     6.111717
94               Tierra del Fuego    38956         1002445  2573.274977